### PR TITLE
Update permissions.yml

### PR DESCRIPTION
**Pull request by Bcrulez**
I think that's how I was supposed to do it. tell me if I missed something

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -1361,19 +1361,18 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
-  beerus:
-    inheritance:
+  beerus:    inheritance:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: inheritance needs to be on its own line

Comment (Bcrulez) [inline PermissionsEx/permissions.yml]: oh ok

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -1361,19 +1361,19 @@ groups:
         - ch.alias.haste4
         - ch.alias.haste5
         - ch.alias.haste6
-  beerus:
-    inheritance:
+  beerus:   
+  inheritance:
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: wrong number of spaces.

Comment (Bcrulez) [inline PermissionsEx/permissions.yml]: oh, ill get right on that

**Pull request merged**